backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/ITUserServiceWrapper.java

```diff
@@ -0,0 +1,32 @@
+package com.redhat.cloud.notifications.recipients.itservice;
+
+import com.redhat.cloud.notifications.recipients.rbac.RbacUser;
+import com.redhat.cloud.notifications.routers.models.Page;
+import io.smallrye.mutiny.Uni;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+@ApplicationScoped
+public class ITUserServiceWrapper {
+
+    @Inject
+    @RestClient
+    ITUserService itUserService;
+
+    public Uni<Page<RbacUser>> getUsers(String accountId, boolean adminsOnly, int i, Integer rbacElementsPerPage) {
+        Path fileName = Path.of("src/main/resources/users.json");
```

Comment:
I don't see this file in this commit. What is it for?

Reply:
It's still a draft :-) this file is currently used to do a static http post. it will be replaced by dynamic content (accountId, adminsOnly and so on) later. I wanted to get it working first.